src/web3/WalletManager.ts

```diff
@@ -0,0 +1,42 @@
+import { Chain } from 'src/ethers/types'
+import { ImportMnemonicAccount } from 'src/web3/KeychainSigner'
+import KeychainAccountManager from 'src/web3/KeychainAccountManager'
+import ValoraCeloWallet from 'src/web3/ValoraCeloWallet'
+import ValoraEthersWallet from 'src/ethers/ValoraEthersWallet'
+
+class WalletManager {
+  keychainAccountManager: KeychainAccountManager
+  contractKitWallet?: ValoraCeloWallet
+  ethersWallets?: Record<Chain, ValoraEthersWallet>
+
+  constructor(private importMnemonicAccount: ImportMnemonicAccount) {
+    this.keychainAccountManager = new KeychainAccountManager()
+  }
+
+  async init() {
+    await this.keychainAccountManager.init(this.importMnemonicAccount)
+    this.contractKitWallet = new ValoraCeloWallet(
+      this.importMnemonicAccount,
+      this.keychainAccountManager
+    )
+    this.ethersWallets = {
+      [Chain.Celo]: new ValoraEthersWallet(Chain.Celo, this.keychainAccountManager),
+    }
+  }
```

Comment:
should the assignment of `this.contractKitWallet` and `this.ethersWallets` be moved to the constructor?

Reply:
Sure, I think that should be fine.